test/karma.conf.js

```diff
@@ -133,36 +134,22 @@ module.exports = function setKarmaConfig(config) {
               envName: 'stable',
             },
           },
-          // transpile 3rd party packages with dependencies in this repository
+          // Transpile dependencies outside this repository with dependencies in this repository
+          // TODO: Remove when the lab will stop exporting components from `@mui/x-tree-view`
           {
             test: /\.(js|mjs|jsx)$/,
-            include:
-              /node_modules(\/|\\)(notistack|@mui(\/|\\)x-data-grid|@mui(\/|\\)x-data-grid-pro|@mui(\/|\\)x-license-pro|@mui(\/|\\)x-data-grid-generator|@mui(\/|\\)x-date-pickers-pro|@mui(\/|\\)x-date-pickers|@mui(\/|\\)x-tree-view)/,
+            include: /node_modules(\/|\\)@mui(\/|\\)x-tree-view/,
             use: {
```

Comment:
We miss notistack here no? https://mui.com/material-ui/react-snackbar/#notistack

Reply:
> This logic is meant to help us test fixes for these components, make sure if we break something in Material UI for MUI X, we know it as soon as we work on the PR

But we don't have any test using those components right?
If the tests pass without the alias.
I can add the alias back, but it will never be used.